client/blocks/reader-related-card-v2/style.scss

```diff
@@ -0,0 +1,431 @@
+// Custom breakpoints needed to match original design
+
+$breakpoint-medium: "( min-width: 661px ) and ( max-width: 730px )";
```

Comment:
Done in: https://github.com/Automattic/wp-calypso/pull/7628/commits/42a6e9679ea21037a721972ab6fa592a07090580